modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContextTest.java

```diff
@@ -392,4 +378,7 @@ public boolean isShow(
 						"contentDashboardItemFilterParameterLabel: value2")));
 	}
 
+	private final ServiceTrackerList<ContentDashboardItemFilterProvider>
```

Comment:
Hey @GabrielHenriquedev, I think you should set this mocked instance into the `ContentDashboardAdminManagementToolbarDisplayContext` created by this unit test. Btw, Has this test passed with this change? Thx